scripts/artifacts.sh

```diff
@@ -1,5 +1,5 @@
-# # Generates the 3 metadata files in `/artifacts` and the polkadot.rs file for integration tests
-#
+# # Generates the 4 metadata files in `/artifacts` and the polkadot.rs file for integration tests.
```

Comment:
Maybe you remove this completely now :rocket:

Reply:
This line or the entire bash script file?